tests/unit/dataactbroker/test_fileRoutes.py

```diff
@@ -60,3 +61,85 @@ def test_list_submissions(file_app, database, user_constants, job_constants):
     g.user = user2
     response = file_app.get("/v1/list_submissions/?certified=mixed")
     assert sub_ids(response) == {submissions[2].submission_id}
+
+
+def test_current_page(file_app, database, user_constants, job_constants, monkeypatch):
+    """Test the route to check what the current progress of the submission is at
+    the correct page
+    """
+
+    cgacs = [CGACFactory() for _ in range(1)]
```

Comment:
If you're only making one you don't really need to do an array. You can just declare a new cgacfactory and work with it without having to index every time.